lms/validation/_bearer_token.py

```diff
@@ -62,19 +62,9 @@ class BearerTokenSchema(marshmallow.Schema):
     oauth_consumer_key = marshmallow.fields.Str(required=True)
     roles = marshmallow.fields.Str(required=True)
 
-    class Meta:
-        """Marshmallow options for this schema."""
-
-        # Silence a strict=False deprecation warning from marshmallow.
-        # TODO: Remove this once we've upgraded to marshmallow 3.
-        strict = True
-
     def __init__(self, request):
-        super().__init__()
-        self._request = request
```

Comment:
We're always accessing the request `self.context["request"]` now, instead of a mix of that and `self._request`. (I'm not sure why `self.context["foo"]` is used instead of `self.foo`, it's a marshmallow pattern.)